grpc/weaviate.proto

```diff
@@ -262,33 +266,33 @@ message ResultAdditionalProps {
 }
 
 message NumberArrayProperties{
```

Comment:
here (and in the following) also we could change `NumberArrayProperties{` to `NumberArrayProperties {`